lib/parse-args.js

```diff
@@ -154,12 +157,6 @@ function buildYargs (withCommands = false) {
     })
     .pkgConf('c8')
     .demandCommand(1)
-    .check((argv) => {
-      if (!argv.tempDirectory) {
-        argv.tempDirectory = resolve(argv.reportsDir, 'tmp')
-      }
-      return true
-    })
     .epilog('visit https://git.io/vHysA for list of available reporters')
```

Comment:
From my understanding, this function should not actually change arguments but only checks to see if they are valid, per the [yargs documentation.](https://yargs.js.org/docs/#api-reference-checkfn-globaltrue)

This condition was never reached per the former code coverage report.